code/modules/mapping/map_template.dm

```diff
@@ -74,7 +74,7 @@
 	if(T.y+height > world.maxy)
 		return
 
-	var/list/bounds = maploader.load_map(file(mappath), T.x, T.y, T.z, cropMap=TRUE)
+	var/list/bounds = maploader.load_map(file(mappath), T.x, T.y, T.z, cropMap=TRUE, no_changeturf=TRUE)
```

Comment:
This is very very very wrong but I'm too tired to explain why

Reply:
The ruins were still working fine but yeah, this likely has side effect I don't know about atm